uber/automated_emails.py

```diff
@@ -1,28 +1,30 @@
-### WARNING - changing the email subject line for an email causes ALL of those emails to be re-sent!
-#   Note that since the EVENT_NAME is used in most of these emails, changing the event name mid-year
-#   could cause literally thousands of emails to be re-sent!
+# WARNING - changing the email subject line for an email causes ALL of those emails to be re-sent!
+# Note that since c.EVENT_NAME is used in most of these emails, changing the event name mid-year
+# could cause literally thousands of emails to be re-sent!
 
 from uber.common import *
 
+
 class AutomatedEmail:
     instances = OrderedDict()
 
```

Comment:
I've been mostly trying to keep ``c.SOME_OPTION`` stuff from getting evaluated at import time, both because it makes unit testing easier (which I'm sure we'll have time for one of these years :/) and because it supports config entries changing at runtime without a server restart.  The latter will be significant if/when we move to something like Redis.